lib/src/hydrated_bloc_delegate.dart

```diff
@@ -14,17 +15,32 @@ class HydratedBlocDelegate extends BlocDelegate {
   final HydratedStorage storage;
 
   /// Builds a new instance of `HydratedBlocDelegate` with the
-  /// default `HydratedBlocStorage`.
-  /// A custom `storageDirectory` can optionally be provided.
+  /// default [HydratedBlocStorage].
+  /// A custom [storageDirectory] can optionally be provided.
   ///
   /// This is the recommended way to use a `HydratedBlocDelegate`.
   /// If you want to customize `HydratedBlocDelegate`
   /// you can extend `HydratedBlocDelegate` and perform the necessary overrides.
+  ///
+  /// [encryptionCipher] is Hive's `HiveCipher`,
+  /// You can provide default one using following snippet:
+  /// ```dart
+  /// import 'package:crypto/crypto.dart';
+  /// import 'package:hive/hive.dart';
+  ///
+  /// const password = 'hydration';
+  /// final byteskey = sha256.convert(utf8.encode(pass)).bytes;
+  /// return HiveAesCipher(byteskey);
+  /// ```
   static Future<HydratedBlocDelegate> build({
     Directory storageDirectory,
+    HiveCipher encryptionCipher,
```

Comment:
I think it would be nice to abstract this so the public api doesn't expose direct dependencies on hive in case we pivot to another storage solution in the future. Thoughts?